packages/polyfills/src/index.ts

```diff
@@ -32,7 +32,7 @@ export const polyfills: {[polyfill: string]: PolyfillDescriptor} = {
 export function mappedPolyfillsForEnv(
   env: 'node' | 'jest' | string[],
 ): {[key: string]: string} {
-  const prefix = `@shopify/polyfills/build/cjs`;
+  const prefix = `@shopify/polyfills`;
```

Comment:
I'm wondering if it would be a terrible thing to just have the CommonJS builds and just have this map to the `build/cjs` instead. Not sure what the impact on treeshaking would be, since this package has `sideEffects` set to `true` anyway.